### ISSUE TITLE
options for formatting HTML

### ISSUE THREAD

**Issue by peterbabic** [UnwrittenFun/prettier-plugin-svelte#66]
Is there a possibility to put trailing angle bracket of long HTML tag on the new line during the formatting, please?

Currently, the output I get is:
```svelte
<a
    class={segment === 'sign-in' ? 'selected' : ''}
    href={$user.id === undefined ? 'sign-in' : 'my-profile'}
    cy="profile">
    {$user.id === undefined ? 'sign-in' : 'my-profile'}
</a>
```

I would prefer it like this (note the trailing > on the newline and the single quotes converted to double quotes):
```svelte
<a
    class={segment === "sign-in" ? "selected" : ""}
    href={$user.id === undefined ? "sign-in" : "my-profile"}
    cy="profile"
>
    {$user.id === undefined ? "sign-in" : "my-profile"}
</a>
```

The prettier playground does it exactly like preffered, see [link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAhgHSgAjzsAG3QGcSBeYEuAcwFsEYdyWcByEgSxqgFpOobHAH521QnDDwAJkKTs2AX2z4cACwBOcAGaUAJAFdqGgHSdpzVgajSdAuBdEdufAXPZ0AnrwAOGiNqcEkoq+GCe5JggfgFBcFEAfKF4wIbGZhYs5DjWtoFQDiJiLvyCOPJsXr7+gcHKUKgA9OgJIAA0IBA+MJzQJMig6Br+AO4ACkMI-SjoAG4Q5u0gAEYa6GAA1nAwAMo+6wI0yDAaBnAdajB0hADqapzwJPtgcDtTD5yzD57I4GRLAmMMDGa3o6GQ2nQhGoHQAViQAB4AITWm22O3QDAAMvYIVCYSB4QidocJABFAwQeB46HnaJDYy-ZboZZwQhLPwCGA3cwwNTIAAcAAYOjFqDc1j5fn44MZZnAlgBHSnwEFdaYgUi8AoOBxLLTKzhaEHoME0gnUOicY6nOlcKA0ckqhVISG0jowFk86R85AAJg9ayChwAwhA6HRwSgoNAFR0jHAACos6ZugmzM4ASRsjB2YA0nG6AEEbDsYJ4JOa4IpFEA)

**Comment by orlov-vo** [comment]
It is related with #21

**Comment by UnwrittenFun** [comment]
I believe this is addressed with the above option, lmk if not

**Comment by rushelex** [comment]
Hello! Thanks for this extension! I restored this ticket because the "svelteBracketNewLine" option does not work. I specified in the .prettierrc file, but it does not change the position of the closing bracket. I also inserted the option "svelteBracketNewLine" in the .prettierrc.js file, and there was also the option of closing the angle bracket without a slash, the result does not change.
![image](https://user-images.githubusercontent.com/41122157/79642279-0c61fa00-81b6-11ea-9478-20c97b206836.png)
